docs/tutorial/part-three/index.md

```diff
@@ -177,18 +180,21 @@ If you go to any of your three pages you'll see the same title added e.g. the
 
 Let's add navigation links to each of your three pages:
 
-```jsx{2-9,12-22}
+`src/components/layout.js`
```

Comment:
Feels like the line `Let's add navigation links to each of your three pages:` might be ambiguous. This line explicitly state we’re modifying Layout component.